operations/configuration.md

```diff
@@ -393,31 +393,33 @@ Returns all tax environments supported by the API.
 | `Value` | number | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
 | `Strategy` | [Tax Rate Strategy](#tax-rate-strategy) | required | Tax strategy type, e.g. relative or flat. |
```

Comment:
unify the casing here `Tax rate strategy`

Reply:
Here I'd put something like `Calculation strategy of the tax rate.` as a description.